guides/type_definitions/input_objects.md

```diff
@@ -52,7 +52,7 @@ class Types::PostAttributes < Types::BaseInputObject
 end
 ```
 
-For a full description of the `argument(...)` method, see the {% internal_link "argument section of the Objects guide","/type_definitions/objects#field-arguments" %}.
+For a full description of the `argument(...)` method, see the {% internal_link "argument section of the Objects guide","/type_definitions/input_objects.html" %}.
```

Comment:
Hmm, this link now points to the same file that the text is in. How about this one instead? 

```suggestion
For a full description of the `argument(...)` method, see the {% internal_link "argument section of the Objects guide","/fields/arguments.html" %}.
```